src/main/res/values/info_strings.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="define_LiferyIconBadge"></string>
```

Comment:
I just want to add that if this is still necessary the name needs to be adjusted :)